### ISSUE TITLE
choco install fails

### ISSUE THREAD

**Issue by rococode** [neovim/neovim#7635]
```Attempt to get headers for https://github.com/neovim/neovim/releases/download/v0.2.1/nvim-win64.zip failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://github.com/neovim/neovim/releases/download/v0.2.1/nvim-win64.zip'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
Downloading neovim 64 bit
  from 'https://github.com/neovim/neovim/releases/download/v0.2.1/nvim-win64.zip'```

Looks like the releases page doesn't actually have a win64 zip for v0.2.1

**Comment by justinmk** [comment]
choco should update to 0.2.0

The download for 0.2.1 was removed because 0.2.2 fixes some important bugs in 0.2.1.

Please report this to the choco maintainer (not us).

**Comment by rococode** [comment]
Okay, I'll do that. Worth noting for others that the --pre flag install works fine
